packages/store-api/src/typeDefs/productGroup.graphql

```diff
@@ -0,0 +1,5 @@
+type StoreProductGroup {
+  hasVariant: [StoreProduct!]!
```

Comment:
Would it make sense for this `hasVariant` to be `null`? 🤔

Reply:
Not really. If a product has no variants, it can just return an empty array like:
```json
{
   "hasVariant": []
}
```

I think this really makes it easier to handle with these data on the frontend

Reply:
Awesome!